### PR TITLE
EN: Update and episode 5 editing

### PR DESCRIPTION
**Pull request by DabirSA**

### CODE REVIEW

File: Story/Episode 5/st_056140.csv [hunk 1]
```diff
@@ -1,11 +1,11 @@
 st_056140_0001#0,"""A.P.241/11/27/10:00"""
 st_056140_0010#0,"""Ah, Aika!"""
-st_056140_0020#0,"""<%me>, Xiera. I heard that I caused you two some trouble. Sorry..."""
-st_056140_0030#0,"""It's no trouble, you weren't in control of what was happening! More importantly are you suffering any adverse effects?"""
+st_056140_0020#0,"""<%me>, Xiera.<%br>I heard that I caused you two some trouble. Sorry..."""
```
Comment (MusouMajo) [inline Story/Episode 5/st_056140.csv]: is that % supposed to be there for something.

File: Story/Episode 5/st_056100.csv [hunk 2]
```diff
@@ -1,15 +1,15 @@
...
-st_056100_0020#0,"""Both the fortresses and the villages on the border<br>were completely devoid of Epic's civilians<br>with only the traces of a hasty evacuation left."""
-st_056100_0030#0,"""As the people were evacuated, the damage to property and<br>harm to them were minimal. However, Margareta was still torn<br>that she had to force her people to evacuate their homes."""
-st_056100_0040#0,"""And yet she remained firm that if the problem were solved,<br>all would be able to be restored to how it was."""
+st_056100_0010#0,"""Guided by its Divine Queen, Margareta, Harriet and the Guardian<br>made their way towards the Holy Kingdom of Epic."""
+st_056100_0020#0,"""Both the fortresses and the villages on the border<br>were completely devoid of Epic's civilians,<br>with only the traces of a hasty evacuation left."""
+st_056100_0030#0,"""As the people had evacuated, the damage to property and<br>harm to them were minimal. However, Margareta was still torn<br>that she had to force her people to evacuate their homes."""
+st_056100_0040#0,"""And yet she remained firm that if the problem were<br>solved, all would be able to be restored to how it was."""
```
Comment (MusouMajo) [inline Story/Episode 5/st_056100.csv]: I don't like this move here, makes more sense to have each line be it's own clause

File: Story/Episode 5/st_056140.csv (hunk 1 above)
Comment (DabirSA) [reply Story/Episode 5/st_056140.csv]: ah whoops that's how es does it

File: Story/Episode 5/st_056100.csv (hunk 2 above)
Comment (DabirSA) [reply Story/Episode 5/st_056100.csv]: I'd agree if the lines were centered, but as they're left-aligned it looks kind of bad if they're all hugely different lengths.

File: UI/quest_counter.csv [hunk 3]
```diff
@@ -333,7 +333,7 @@ Quest_00020130#0,"""- Clear the Free Field \""Shironia Exploration\"""""
 Quest_00020140#0,"""- Clear the Free Field \""Facility Exploration\"""""
 Quest_00020150#0,"""- Clear the Free Field \""Shironia Exploration\"""""
 Quest_00020160#0,"""- Clear the Free Field \""Tokyo Exploration\"""""
-Quest_00020170#0,"""Complete one of the following:<br>- Clear the Free Field \""Tokyo Exploration\""<br>- Clear the Buster Quest \""Battle the Devil Castle\""<br>- Clear the Emergency Quest \""Endless Belligerence\"""""
+Quest_00020170#0,"""Complete one of the following:<br>- Clear the Free Field \""Tokyo Exploration\""<br>- Clear the Buster Quest \""Battle the Devil Castle\""<br>- Clear the Emergency Quest \""Endless Belligerence\"""<br>- Clear the Emergency Quest \""Perpetual Madness\"" (Releases September 26)""
```
Comment (alama) [inline UI/quest_counter.csv]: quote misplace

**Pull request merged**